api/turing/service/router_deployment_service.go

```diff
@@ -480,8 +482,23 @@ func deleteK8sService(
 	controller cluster.Controller,
 	service *cluster.KubernetesService,
 	timeout time.Duration,
+	isCleanUp bool,
 ) error {
-	return controller.DeleteKubernetesService(context.Background(), service.Name, service.Namespace, timeout)
+	ctx := context.Background()
+	var err error
+	if isCleanUp {
```

Comment:
**Nit:** This "if-else" block can be re-written concisely as:

```golang
controller.DeleteKubernetesDeployment(ctx, service.Name, service.Namespace, timeout, isCleanUp)
```

Same comment for the block at the end of this method too. It can be:
```golang
return controller.DeleteKubernetesService(ctx, service.Name, service.Namespace, timeout, isCleanup)
```

There are a couple other such instances - something that was overlooked in the original MR. Anyway, I'll include this minor change in my next MR to Turing.